spring-cloud-netflix-eureka-client/src/test/java/org/springframework/cloud/netflix/eureka/EurekaHealthCheckHandlerTests.java

```diff
@@ -72,7 +76,35 @@ public void testDown() throws Exception {
 
 		initialize(UpHealthConfiguration.class, DownHealthConfiguration.class);
 
-		InstanceStatus status = healthCheckHandler.getStatus(InstanceStatus.UNKNOWN);
+		InstanceStatus status = this.healthCheckHandler.getStatus(InstanceStatus.UNKNOWN);
+		assertThat(status).isEqualTo(InstanceStatus.DOWN);
+	}
+
+	@Test
+	public void testDown2() throws Exception {
+
```

Comment:
Please remove the unnecessary empty lines at the beginning of the test. You can add dividing line between the `act` statement, where `status` is retrieved and the assertion.

Please add more meaningful names to tests. For example `testReturnDownWhenBlockingIndicatorsUpAndReactiveDown` is a better name than `testDown2`.